MmSupervisorPkg/Core/MmSupervisorCore.inf

```diff
@@ -19,16 +19,16 @@
   BASE_NAME                      = MmSupervisorCore
   FILE_GUID                      = 4E4C89DC-A452-4B6B-B183-F16A2A223733
   MODULE_TYPE                    = MM_CORE_STANDALONE
-  VERSION_STRING                 = 11.001
+  VERSION_STRING                 = 13.001

```

Comment:
It would be helpful to call out the jump from `11` to `13` in the release notes since they're there to make understanding things like that easier when picking up the latest release.

Reply:
Updated PR description. Will update in release notes as well.